lib/elixir/lib/kernel.ex

```diff
@@ -2826,17 +2826,25 @@ defmodule Kernel do
   it is not loaded. Check `Code.ensure_loaded/1` for more
   information.
 
+  If `module` is an Erlang module (as opposed to an Elixir module), this
+  function always returns `false`.
+
   ## Examples
 
       iex> macro_exported?(Kernel, :use, 2)
       true
 
+      iex> macro_exported?(:erlang, :abs, 1)
+      false
+
   """
   @spec macro_exported?(atom, atom, integer) :: boolean
   def macro_exported?(module, macro, arity) do
     case :code.is_loaded(module) do
-      {:file, _} -> :lists.member({macro, arity}, module.__info__(:macros))
-      _ -> false
+      {:file, _filename} ->
```

Comment:
After another thought I think we should use `function_exported?/2` here, as it makes `macro_exported?` to behave in the similar way as `function_exported?` for not loaded modules.